ccm_web/server/src/user/user.errors.ts

```diff
@@ -0,0 +1,12 @@
+export class UserNotFoundError extends Error {
+  date: Date
+  public name = 'UserNotFoundError'
+
+  constructor (userLoginId: string) {
+    super(
+      `The user "${userLoginId}" could not be found; ` +
+      'the database may have been modified or there may be a problem with the application code.'
+    )
+    this.date = new Date()
+  }
+}
```

Comment:
I appreciate your effort with structuring the custom error types here, this really make code well organized. Thanks for taking time and doing it!

Reply:
😃